src/shared/copy/notifications/categories/accounts-users-orgs.ts

```diff
@@ -91,16 +91,6 @@ export const orgCreateSuccess = (): Notification => ({
   message: 'Organization was successfully created',
 })
 
-export const orgDefaultSettingError = (orgName: string): Notification => ({
```

Comment:
We don't need these notifications, since we decided to go with a single 'user successfully updated preferences' notification.